src/plugins/workspace/public/components/workspace_form/workspace_form.tsx

```diff
@@ -176,22 +177,27 @@ export const WorkspaceForm = (props: WorkspaceFormProps) => {
         </EuiPanel>
       )}
       <EuiSpacer />
-      <EuiPanel>
-        <EuiTitle size="s">
-          <h2>
-            {i18n.translate('workspace.form.selectDataSource.title', {
-              defaultMessage: 'Select Data Sources',
-            })}
-          </h2>
-        </EuiTitle>
-        <SelectDataSourcePanel
-          errors={formErrors.selectedDataSources}
-          onChange={setSelectedDataSources}
-          savedObjects={savedObjects}
-          selectedDataSources={formData.selectedDataSources}
-          data-test-subj={`workspaceForm-dataSourcePanel`}
-        />
-      </EuiPanel>
+
+      {/* SelectDataSourcePanel is only visible for dashboard admin */}
+      {isDashboardAdmin && (
```

Comment:
Do we need to detect if data source is enabled?

Reply:
When data source is disabled, the list will be empty. Do you think we should hide this panel if data source is disabled?

Reply:
I think we should, when data source is not enabled, there is not even a data source concept right?

Reply:
Updated.